distribution/CHANGELOG.md

```diff
@@ -0,0 +1,25 @@
+# Emerge Reaper Changelog
```

Comment:
probably didn't mean to have this say reaper?

Reply:
thanks! done.